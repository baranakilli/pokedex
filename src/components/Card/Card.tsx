import { Robot } from "../../actions";

const Card = (props: Robot) => {
   const { id, name, email } = props;
   return(
      <div className="bg-light-blue dib br3 pa1 pv3 ma2 grow ba bw1 b--blue shadow-5">
         <img alt="robots" width={'200px'} height={'200px'} src={`https://robohash.org/${id}?size=200x200`} />
         <div style={{width: '300px'}}>
            <h2>{name}</h2>
            <p>{email}</p>
         </div>
      </div>
   );
}

export default Card;