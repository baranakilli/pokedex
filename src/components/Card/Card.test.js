import { shallow } from 'enzyme';
import Card from './Card';

it('expect to render Card component', () => {
  // console.log(shallow(<Card />));
  expect(shallow(<Card />)).toMatchSnapshot();
});
