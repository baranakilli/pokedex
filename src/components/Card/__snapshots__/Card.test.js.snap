// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card component 1`] = `
<div
  className="bg-washed-yellow  br3 ma2 ba bw2 b--red shadow-5 inline-flex flex-column items-center"
>
  <img
    alt="pokemon undefined"
    className="pv3"
    height="200px"
    src=""
    width="200px"
  />
  <div
    style={
      Object {
        "width": "250px",
      }
    }
  >
    <h2
      className="pokemon-title ttc white ma0 pb3 bg-red h-100"
    />
    <div
      className="line"
    >
      <div
        className="circle"
      >
        <div
          className="inner-circle"
        />
      </div>
    </div>
    <p
      className="black pb2 pt3 fw6 mt0 mb2 bg-white"
    >
      Weight: 
      kg
      <br />
      Height: 
      0 cm
    </p>
  </div>
</div>
`;
